sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/ResolveDefaultColumns.scala

```diff
@@ -280,6 +280,9 @@ case class ResolveDefaultColumns(
       case SubqueryAlias(_, r: UnresolvedCatalogRelation) =>
         StructType(r.tableMeta.schema.fields.dropRight(
           enclosingInsert.get.partitionSpec.size))
+      case SubqueryAlias(_, r: View) =>
```

Comment:
I'm a bit confused about what it means to `INSERT INTO` a view. Can you leave a comment